task-3/results/k8s/secret.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: data-export-secret
+  namespace: default
+type: Opaque
+stringData:
+  DB_USER: "postgres"
+  DB_PASSWORD: "postgres123"
+  S3_ACCESS_KEY: "minioadmin"
+  S3_SECRET_KEY: "minioadmin123"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)